src/amplitude.js

```diff
@@ -47,10 +47,10 @@ define(function (require) {
    *
    *  </code></div>
    */
-  p5.Amplitude = function(smoothing) {
+  p5.Amplitude = function(smoothing, bufferSize = 2048) {
 
     // Set to 2048 for now. In future iterations, this should be inherited or parsed from p5sound's default
-    this.bufferSize = safeBufferSize(2048);
+    this.bufferSize = safeBufferSize(bufferSize);
```

Comment:
the comment above this line should be deleted or updated. I'm curious if you considered the other options mentioned in the comment?

Reply:
@therewasaguy  
actually comment must be modified  :smile:
 and
as far as inheritance part is concerned i think , i need to modify p5.sound pass a optional argument "bufferSize " like i did in here , but right now let's leave it and i will make it another pr !
what do you say ?